helm/portieris/values.yaml

```diff
@@ -15,6 +15,9 @@ service:
   port: 443
   targetPort: 8000
 
+# If not running on IBM Cloud Container Service set to false
+ibmContainerService: true
```

Comment:
Are there rules on the case of Helm values? Would it be an antipattern for `IBM` here to be capitalized? `IBMContainerService` looks better imo.